bin/newman.js

```diff
@@ -379,6 +380,84 @@ var _ = require('lodash'),
         return optionsObj;
     },
 
+    /**
+     * Moves the item from one index to another in an array, shifting subsequent items.
+     *
+     * @param {Array} arr - The array of items.
+     * @param {Number} fromIndex - The position from which item is to be moved.
+     * @param {Number} toIndex - The position to which item is to be moved
+     * @returns {Array} - The array with the modified positions.
+     */
+    move = function (arr, fromIndex, toIndex) {
+        arr.splice(toIndex, 0, arr.splice(fromIndex, 1)[0]);
+        return arr;
+    },
+
+    /**
+     * Moves all the boolean flags to the end of the options list.
+     *
+     * @param {Array} arr - The array of supplied CLI options.
+     * @returns {Array} - The modified array with all boolean options at the end.
+     */
+    updateBooleanFlags = function (arr) {
+        var booleanFlags = [
```

Comment:
Not a good idea. Now we have a second set of enumerations to look after and adds dual source of truth. Making code difficult to maintain.